modules/rosa-upgrading-manual-ocm.adoc

```diff
@@ -28,7 +28,11 @@ endif::sts[]
 . Log in to {cluster-manager-url}.
 . Select a cluster to upgrade.
 . Click the *Settings* tab.
-. In the *Update strategy* pane, click *Manual* or *Individual Updates*.
+. In the *Update strategy* pane, select *Individual Updates*.
+. Select the version you want to upgrade your cluster to. Recommended cluster upgrades are notated in the UI.
```

Comment:
Recommend changing to "Recommended cluster upgrades appear in the UI." for simpler language.

Reply:
Made the change.